bundle-workflow/src/test_workflow/dependency_installer.py

```diff
@@ -0,0 +1,62 @@
+import os
+import shutil
+
+from s3_utility import S3ReadWrite
+
+
+class DependencyInstaller:
+    """
+    Provides a dependency installer for the test suites.
+    """
+
+    def __init__(self, build):
+        self.build_id = build.id
+        self.version = build.version
+        self.arch = build.architecture
+        self.s3_utility = S3ReadWrite()
+        self.s3_maven_location = (
+            f"/builds/{self.version}/{self.build_id}/{self.arch}/maven"
+        )
+        self.s3_build_location = (
+            f"/builds/{self.version}/{self.build_id}/{self.arch}/plugins"
+        )
```

Comment:
nit: this location provider logic is common across build and test workflows and should be read from a single place. Check if there's already a utility/or we can make it. It's okay if you create an issue and take it separately outside this PR.